native/ipc.ts

```diff
@@ -10,26 +13,27 @@ export const OPEN_PATH = "IPC_OPEN_PATH";
 //
 // The workaround is to use the electron native open dialog. As a bonus we
 // can configure it to allow users to create new directories.
-export const getDirectoryPath = () =>
+export const getDirectoryPath = (): void =>
```

Comment:
```suggestion
export const getDirectoryPath = (): string =>
```

It returns the file path that is chosen within the file dialog.

Reply:
Fixed in https://github.com/radicle-dev/radicle-upstream/pull/1193/commits/79e2b4b5d2cab8a3bc660aab9f9e7ee8a4522c5f